docs/how-to-build.md

```diff
@@ -21,31 +22,241 @@ The project itself includes the source code of several 3rd-party libraries, whic
   - libunwind
   - ncurses
   - readline
-  - bzip2
-  - doubleconversion
-  - fatal
-  - fbthrift
-  - folly
-  - gflags
-  - glog
-  - googletest
-  - jemalloc
-  - libevent
-  - mstch
-  - proxygen
-  - rocksdb
-  - snappy
-  - wangle
-  - zlib
-  - zstd
-
-### How to get 3rd-party Libraries
+
+#### Build and install 3rd-party Libraries
 [Install guide](https://github.com/vesoft-inc/nebula-3rdparty/blob/master/README.md)
 
 
-### How to build
+#### Build nebula
   - 1. create build dir : `cd nebula && mkdir build && cd build`
   - 2. generate makefile : `cmake -DNEBULA_THIRDPARTY_ROOT=${3rd-party_install_root} ..`
   - 3. make : `make or make -j${threadnum}`
   - 4. install: `make install`
   - notes : the default installation dir is **/usr/local/**, if you want to change the dir, on step 2, your command can be `cmake -DCMAKE_INSTALL_PREFIX=$your_nebula_install_dir ..`
+
```

Comment:
if you preview the markdown file, you will notice the `step 2` displays as `ii`, so, make them inconsitent